### PR TITLE
feature: Add Release note for PMD Legacy removal CY-5203

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

**Comment by prcr** [comment]
Here's the live preview of the announcement page:

https://deploy-preview-1047--docs-codacy.netlify.app/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal/

**Comment by lventura-codacy** [comment]
> Here's the live preview of the announcement page:
> 
> https://deploy-preview-1047--docs-codacy.netlify.app/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal/

@prcr thanks for always including these, makes our lives easier 🙏

File: docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Removal of PMD (Legacy) February 16, 2022
+
+On the week of February 16th 2022 we'll be removing the tool **PMD (Legacy)** from Codacy.
```
Comment (pedrocodacy) [inline docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md:8]:
```suggestion
On the week of February 16th 2022 Codacy will stop supporting the **PMD (Legacy)** linter.
```

Comment (pedrocodacy) [reply docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md:8]: dunno if linter is better than tool though, probably @prcr will be able to help here

Comment (prcr) [reply docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md:8]: Throughout the docs we consistently refer to "tools", for example:

- https://docs.codacy.com/getting-started/supported-languages-and-tools/
- https://docs.codacy.com/related-tools/local-analysis/client-side-tools/

Would you say that "linter" is more correct, or more specific perhaps? We can always consider adjusting the wording used on the docs, although I would do it as a separate task. :slightly_smiling_face:

**Review by stefanvacareanu7 — APPROVED** [review]

File: docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+The **PMD (Legacy)** tool runs [PMD 5.8.1](https://github.com/pmd/pmd/releases/tag/pmd_releases%2F5.8.1){: target="_blank"}, and was introduced to allow a smoother migration to PMD 6.
+
+However, the legacy version of PMD is almost five years old and is no longer being maintained by the tool developers, while PMD 6 is now very stable and provides newer features.
+
+## If you were using PMD (Legacy)
+
+To continue using PMD to analyze your repositories, [enable the **PMD** tool](../../repositories-configure/configuring-code-patterns.md) on the **Code patterns** page of your repositories.
```
Comment (prcr) [inline docs/release-notes/cloud/cloud-2022-02-16-pmd-legacy-removal.md:16]: Can we also include here a way to let customers reach Support?

```suggestion
To continue using PMD to analyze your repositories, [enable the **PMD** tool](../../repositories-configure/configuring-code-patterns.md) on the **Code patterns** page of your repositories.

If you have any questions or need help, please contact <mailto:support@codacy.com>.
```

**Review by prcr — APPROVED** [review]

**Pull request merged**